lib/Event/ReorderEvent.php

```diff
@@ -30,7 +32,7 @@ class ReorderEvent extends AbstractDocumentEvent
      */
     public function __construct($document, $destId, $after)
     {
-        parent::__construct($document);
+        $this->document = $document;
```

Comment:
Because I also changed the class it extends from, and the new one doesn't have a constructor. Don't like this inheritance structure at all, would prefer to replace that with interfaces and traits, but this seemed to be the cleanest solution for now.